### DISCUSSION TITLE
Karazhan loot problem

### DISCUSSION THREAD

**Discussion by Lux881** [chromiecraft/chromiecraft#6406] [General]
Good morning, I completed a run of Kara and encountered a problem with the Boss Chess loot.
The Heart Flame Leggings mail doesn't let me wear them and gives me the warning "you cannot equip more than of those" - Plus it doesn't make me budge completely.
How do I solve it?
![WoWScrnShot_120823_045034](https://github.com/chromiecraft/chromiecraft/assets/153253813/b34f4a2c-ec70-41b1-b052-4962dd307001)

**Comment by BeefSteaker** [comment]
Hi, do you have one of the unique equipped dungeon gems on those legs. If you have another on any equipped piece of gear you won't be able to use both. 

Remove either of the unique gems and you should be able to equip the legs

**Reply by BeefSteaker** [reply]
Shining Fire Opal I believe is the issue
